### ISSUE TITLE
Create a test API endpoint that proves Catapult can contact the OpenShift API

### ISSUE THREAD

**Issue by ALRubinger** [redhat-kontinuity/catapult#117]
To be used in verification during CI; currently we simply ask if Catapult has an authenticated GitHub user, but this doesn't test the Catapult > OpenShift communication layer.

FYI @sabre1041

**Comment by ALRubinger** [comment]
Part of health checks?  @gastaldi what do you think?

**Comment by gastaldi** [comment]
Agreed, we could have a Health check endpoint that returns whether the OpenShift API is reachable or not
